trade-finance-manager-api/src/v1/api.js

```diff
@@ -583,6 +583,23 @@ const updatePortalGefDealStatus = async (dealId, status) => {
   }
 };
 
+const addUnderwriterCommentToGefDeal = async (dealId, commentType, comment) => {
+  try {
+    const response = await axios({
+      method: 'post',
+      url: `${centralApiUrl}/v1/portal/gef/deals/${dealId}/comment`,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      data: { dealId, commentType, comment },
+    });
+
+    return response.data;
+  } catch ({ response }) {
+    return false;
```

Comment:
Can we please add console.error({ response }) for logging purposes.

Reply:
Good spot. I've fixed this now